server/routes/barcode/contacts/CreateContactByPrisonNumberController.ts

```diff
@@ -55,7 +55,13 @@ export default class CreateContactByPrisonNumberController {
     recipientForm.prisonId = createNewContactByPrisonNumberForm.prisonId
     try {
       const { prisonNumber, prisonId, prisonerName } = recipientForm
-      await this.contactService.createContact(req.session.slmToken, prisonerName, prisonId, prisonNumber)
+      const contact = await this.contactService.createContact(
```

Comment:
When creating the Contact, we now need to capture the ID for the new contact and put it in the recipientForm.
Both controllers do this.